stubs/inertia/app/Http/Middleware/HandleInertiaRequests.php

```diff
@@ -21,9 +21,17 @@ class HandleInertiaRequests extends Middleware
      * @param  \Illuminate\Http\Request  $request
      * @return string|null
      */
-    public function version(Request $request)
+    public function version(Request $request): ?string
```

Comment:
Please remove this return type.

Reply:
I only added these for consistency with the parent method. I have updated the PR.